libs/feature/dataviz/src/lib/chart-view/chart-view.component.html

```diff
@@ -1,23 +1,25 @@
 <div class="w-full h-full flex flex-col">
   <div
-    class="flex flex-col space-y-2 sm:flex-row sm:space-y-0 sm:space-x-2 justify-between text-[13px]"
+    class="flex flex-col flex-wrap space-y-2 sm:flex-row sm:space-y-0 sm:space-x-2 justify-between text-[13px] gap-4"
```

Comment:
In principle, when modifying a presentation component, we shouldn't have to modify anything that uses this component. Otherwise this indicates that the encapsulation is broken and that our changes are leaking "outside" of the modified component scope.

In that case I think it is acceptable to adjust options because the dropdown is a little bit larger, but modifying the layout (introducing `flex-wrap` etc.) is a too big change in my opinion.